app/search_builders/curation_concerns/filter_suppressed_with_roles.rb

```diff
@@ -10,8 +10,9 @@ module FilterSuppressedWithRoles
     include CurationConcerns::FilterSuppressed
 
     def only_active_works(solr_parameters)
-      # Do not filter works if current user is permitted to take workflow actions
-      return if user_has_active_workflow_role?
+      # Skip the filter if work is not suppressed or if the current user
+      # is permitted to take workflow actions
+      return if work_not_suppressed? || user_has_active_workflow_role?
```

Comment:
is the `work_not_suppressed?` check required here?  Won't the super just take care of this?